### DISCUSSION TITLE
Is it possible to only search for a specific tracker's data?

### DISCUSSION THREAD

**Discussion by ghost** [cross-seed/cross-seed#208] [Q&A] [answered]
For instance restrict the search to RED or OPS torrent data in your `torrentDir` — it's not optimal to crawl through your entire torrent directory when cross-seeding between dedicated music trackers.

I could temporarily move the RED/OPS torrent data to another location, but that's an additional step and in my use  at least rtorrent/rutorrent have handled moving large amount of torrent data poorly (crashed mid-operation).

**Comment by mmgoodnow** [comment, marked as answer]
No, that is not currently possible. My recommendation is to copy, rather than move, the relevant torrent files to a different directory and use that as your torrentDir.

Edit: pasting a solution from below

> It might sound weird but you can actually use `grep` to filter torrents by announce url. 
> 
> https://stackoverflow.com/questions/16956810/how-do-i-find-all-files-containing-specific-text-on-linux

**Reply by ghost** [reply]
> No, that is not currently possible. My recommendation is to copy, rather than move, the relevant torrent files to a different directory and use that as your torrentDir.

Can you consider this as a feature request?

I don't know how to copy the data of a tracker's torrents (thousands) without the assist of some script... if `cross-seed` handed this it'd be splendid and would surely make your program the de-facto standard.

**Reply by mmgoodnow** [reply]
It might sound weird but you can actually use `grep` to filter torrents by announce url. 

https://stackoverflow.com/questions/16956810/how-do-i-find-all-files-containing-specific-text-on-linux

**Reply by mmgoodnow** [reply]
Advanced filtering is something I don't plan on supporting from within cross-seed at the moment, though there are a couple existing feature requests for it.

**Reply by ghost** [reply]
> It might sound weird but you can actually use `grep` to filter torrents by announce url.
> 
> https://stackoverflow.com/questions/16956810/how-do-i-find-all-files-containing-specific-text-on-linux

Fantastic, this appears to work (for Transmission):

`grep -rl ~/.config/transmission/torrents/ -e <the announce domain>`

**UPDATE**: `grep -rl ~/.config/transmission/torrents/ -e <the announce domain> | xargs cp -t ~/Documents/RED` finished the entire process, then point `cross-seed` to the path and save torrents to `~/Documents/OPS` for instance.
